packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.html

```diff
@@ -1,65 +1,95 @@
+          >
+            <lightning-icon
+              class="slds-current-color slds-m-right_xx-small"
+              icon-name="utility:close"
+              alternative-text={labels.close}
+            ></lightning-icon>
           </button>
-          <div id="quickview-modal-heading"
-            class="slds-modal__title slds-truncate slds-grid slds-grid_vertical-align-center">
-            <div class="slds-grid slds-truncate slds-grid_vertical-align-center slds-text-align_left">
+          <div
+            id="quickview-modal-heading"
+            class="slds-modal__title slds-truncate slds-grid slds-grid_vertical-align-center"
+          >
+            <div
+              class="slds-grid slds-truncate slds-grid_vertical-align-center slds-text-align_left"
+            >
               <div class="slds-m-right_small">
-                <c-quantic-result-label result={result} icon-only></c-quantic-result-label>
+                <c-quantic-result-label
+                  result={result}
+                  icon-only
+                ></c-quantic-result-label>
               </div>
               <div class="slds-truncate quickview__result-title">
-                <c-quantic-result-link target="_blank" result={result} engine-id={engineId}></c-quantic-result-link>
+                <c-quantic-result-link
+                  target="_blank"
+                  result={result}
+                  engine-id={engineId}
+                ></c-quantic-result-link>
               </div>
             </div>
-            <div class="slds-text-align_right quickview__result-date slds-col_bump-left">
-              <lightning-formatted-date-time value={result.raw.date}></lightning-formatted-date-time>
+            <div
+              class="slds-text-align_right quickview__result-date slds-col_bump-left"
+            >
+              <lightning-formatted-date-time
+                value={result.raw.date}
+              ></lightning-formatted-date-time>
             </div>
           </div>
         </header>
-        <template if:true={isLoading}>
-          <div onclick={stopPropagation}
-            class="quickview__spinner-container slds-modal__content slds-p-around_large slds-is-relative">
-            <lightning-spinner alternative-text="Loading" size="large"></lightning-spinner>
-          </div>
-        </template>
-        <iframe id="quickview__content-container"
-          class="quickview__content-container slds-modal__content slds-p-around_large slds-wrap" src={contentURL}
-          height="100%" onload={onIframeLoaded}></iframe>
-        <template if:false={isLoading}>
-          <footer onclick={stopPropagation} class="slds-modal__footer slds-p-top_none slds-p-bottom_none">
-            <slot name="footer"></slot>
-          </footer>
-        </template>
+        <c-quantic-quickview-content
+          id="quickview__content-container"
+          class="quickview__content-container slds-modal__content slds-wrap"
```

Comment:
Are these classes really affecting the look of the `c-quantic-quickview-content` component?

Reply:
yes they are needed, for example if I remove the slds-modal__content it removes the background for the modal where the content is